<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DlgStartUp</class>
 <widget class="QDialog" name="DlgStartUp">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>279</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Start Up</string>
  </property>
  <widget class="QPushButton" name="pushBtnMaskEdit">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>30</y>
     <width>201</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Launch Mask Editor and Cleaner</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushBtnSplitCmp">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>110</y>
     <width>201</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Launch Split View Compare</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>190</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Note: You can get to either one of these
options from the tools menu.</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBoxShow">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>250</y>
     <width>141</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Always show on startup</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
